docs/using-the-jdbc-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -23,14 +23,14 @@ To enable the Limitless Connection Plugin, add the plugin code `limitless` to th
+|----------------------------------------------------|:-------:|:--------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|---------------|
+| `limitlessTransactionRouterMonitorIntervalMs`      | Integer |    No    | This property is the interval in milliseconds, that the plugin polls the database for available transaction routers and their load metrics. A lower value will increase the frequency of polling, and a higher value will decrease the frequency of polling. <br><br>The default value of 15 seconds was chosen to match the expected refresh rate of load metric metadata from the database.<br><br>Note that there will always be a delay between when the database updates its load metric info and when the Limitless Connection Plugin polls for it. If your Limitless database experiences fluctuating load between transaction routers, you may want to consider lowering `limitlessTransactionRouterMonitorIntervalMs` to reduce this delay and ensure the Limitless Connection Plugin load balancing has fresher info to work with.                                                                                                                                                    | `15000`       | `30000`       |
```

Comment:
Thought we discussed using 7 seconds ?

Reply:
Other than that it looks good

Reply:
I plan on making the change to the default value in a separate branch/commit